docs/guides/experimental-exporter.md

```diff
@@ -163,7 +163,7 @@ For security reasons, [databricks_secret](../resources/secret.md) cannot contain
 To speed up export, Terraform Exporter performs many operations, such as listing & actual data exporting, in parallel using Goroutines.  Built-in defaults control the parallelism, but it's also possible to tune some parameters using environment variables specific to the exporter:
 
 * `EXPORTER_WS_LIST_PARALLELISM` (default: `5`) controls how many Goroutines are used to perform parallel listing of Databricks Workspace objects (notebooks, directories, workspace files, ...).
-* `EXPORTER_DIRECTORIES_CHANNEL_SIZE` (default: `100000`) controls the channel's capacity when listing workspace objects. Please ensure that this value is big enough (greater than the number of directories in the workspace; default value should be ok for most cases); otherwise, there is a chance of deadlock.
+* `EXPORTER_DIRECTORIES_CHANNEL_SIZE` (default: `300000`) controls the channel's capacity when listing workspace objects. Please ensure that this value is big enough (greater than the number of directories in the workspace; default value should be ok for most cases); otherwise, there is a chance of deadlock.
```

Comment:
Curious -- is there some specific reason this was increased 3x?

Reply:
I was testing on our field-eng workspace when I realized that we could get a deadlock :-( It's not bad from memory consumption, so I decided to raise a default value.

Reply:
I see, sounds good.